cypress/integration/queryBuilder.spec.js

```diff
@@ -0,0 +1,32 @@
+describe('test query builder', () => {
+  beforeEach(() => {
+    cy.server();
+
+    cy.route('GET', 'api/repository/v1/datasets/**').as('getDataset');
```

Comment:
What does `**` do here

Reply:
it's a wildcard in the url, since the uuid in this request is different in my local env vs. what's running in integration